src/components/ProjectSearch.js

```diff
@@ -118,8 +124,16 @@ export class ProjectSearch extends Component<Props, State> {
     }
   }
 
-  doSearch(searchTerm: string) {
-    this.props.searchSources(searchTerm);
+  getModifiers() {
```

Comment:
If you change this to `getModifiers = () => {  }`, you can avoid binding on line 98.